### PR TITLE
en,zh: add 1.4.0-beta.1 release notes

### PR DESCRIPTION
**Pull request by WizardXiao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.4.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# TiDB Operator 1.4.0-beta.1 Release Notes
+
+Release date: October 27, 2022
+
+TiDB Operator version: 1.4.0-beta.1
+
+## New Feature
+
+- TiDB Operator supports AWS EBS snapshot Backup and Restore ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). the feature has the advantages:
```
Comment (shichun-0415) [inline en/releases/release-1.4.0-beta.1.md:13]:
```suggestion
- TiDB Operator supports AWS EBS snapshot Backup and Restore ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). This feature has the following advantages:
```

Comment (ran-huang) [inline en/releases/release-1.4.0-beta.1.md:13]:
```suggestion
- TiDB Operator supports snapshot backup and restore based on Amazon EBS ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). This feature has the following benefits:

```

File: en/releases/release-1.4.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    - Reduce backup impact (qps < 5%)
```
Comment (ran-huang) [inline en/releases/release-1.4.0-beta.1.md:14]:
```suggestion
    - Reduce backup impact (QPS < 5%)
```

File: zh/releases/release-1.4.0-beta.1.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# TiDB Operator 1.4.0-beta.1 Release Notes
+
+发布日期：2022 年 10 月 27 日
+
+TiDB Operator 版本：1.4.0-beta.1
+
+## 新功能
+
+- 支持基于 AWS EBS 的 TiDB 集群 volume-snapshot 的备份和恢复 ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky))，此功能有以下特点：
```
Comment (ran-huang) [inline zh/releases/release-1.4.0-beta.1.md:13]: 这个官方正式的名称应该是 Amazon EBS (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html)
```suggestion
- 支持基于 Amazon EBS 的 TiDB 集群 volume-snapshot 的备份和恢复 ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky))，此功能有以下特点：
```

File: zh/releases/release-1.4.0-beta.1.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    - 最小化备份影响 QPS < 3%
```
Comment (ran-huang) [inline zh/releases/release-1.4.0-beta.1.md:14]: 英文写的是 5% ？

File: en/releases/release-1.4.0-beta.1.md (hunk 2 above)
Comment (shichun-0415) [inline en/releases/release-1.4.0-beta.1.md:14]:
```suggestion
    - Reduce backup impact (QPS < 5%)
```

File: zh/releases/release-1.4.0-beta.1.md (hunk 4 above)
Comment (shichun-0415) [reply zh/releases/release-1.4.0-beta.1.md:14]: 用户文档写的 5%：https://github.com/pingcap/docs-tidb-operator/pull/2049/files#diff-34ed61b443f8824df3f39afe5c9fcec62f1147b7e5a0c8be44c92efd75762f07R18

Comment (shichun-0415) [inline zh/releases/release-1.4.0-beta.1.md:14]:
```suggestion
    - 降低备份影响至最低：QPS < 5%
```

Comment (shichun-0415) [reply zh/releases/release-1.4.0-beta.1.md:14]: @fengou1 PTAL

Comment (shichun-0415) [reply zh/releases/release-1.4.0-beta.1.md:14]: From Ou Feng: 实际测试影响为 0， 保守起见，我们统一3%吧。

File: en/releases/release-1.4.0-beta.1.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+Release date: October 27, 2022
+
+TiDB Operator version: 1.4.0-beta.1
+
+## New Feature
+
+- TiDB Operator supports snapshot backup and restore based on Amazon EBS ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). This feature has the following benefits:
+
+    - Reduce backup impact (QPS < 3%)
```
Comment (shichun-0415) [inline en/releases/release-1.4.0-beta.1.md:15]:
```suggestion
    - Reduce backup impact (QPS < 5%)
```

File: en/releases/release-1.4.0-beta.1.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+Release date: October 27, 2022
+
+TiDB Operator version: 1.4.0-beta.1
+
+## New Feature
+
+- TiDB Operator supports snapshot backup and restore based on Amazon EBS ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). This feature has the following benefits:
+
+    - Reduce backup impact (QPS < 5%)
```
Comment (ran-huang) [inline en/releases/release-1.4.0-beta.1.md:15]:
```suggestion
    - Reduce the impact of backup on QPS to less than 5%
```

File: zh/releases/release-1.4.0-beta.1.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+发布日期：2022 年 10 月 27 日
+
+TiDB Operator 版本：1.4.0-beta.1
+
+## 新功能
+
+- 支持基于 Amazon EBS 的 TiDB 集群 volume-snapshot 的备份和恢复 ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky))，此功能有以下特点：
+
+    - 降低备份影响至最低：QPS < 5%
```
Comment (ran-huang) [inline zh/releases/release-1.4.0-beta.1.md:15]: @shichun-0415 @fengou1 可以改成这样吗
```suggestion
    - 将备份对 QPS 的影响降至小于 5%
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**